cookbook/configuration/pdo_session_storage.rst

```diff
@@ -172,10 +229,10 @@ following (MySQL):
 .. code-block:: sql
 
     CREATE TABLE `session` (
-        `session_id` VARBINARY(128) NOT NULL PRIMARY KEY,
-        `session_data` BLOB NOT NULL,
-        `session_time` INTEGER UNSIGNED NOT NULL,
-        `session_lifetime` MEDIUMINT NOT NULL
+        `sess_id` VARBINARY(128) NOT NULL PRIMARY KEY,
+        `sess_data` BLOB NOT NULL,
```

Comment:
Isn't this to specific to document?

Anyway, I find the proposed text of @javiereguiluz way to long for something as minor and not that related as this. If we want to add something (I'm not sure yet), I would reduce it to just something as "If `BLOB` can't store large enough data for you, consider user the `MEDIUMBLOB` type."